components/post_view/date_separator/index.js

```diff
@@ -1,30 +1,4 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import {connect} from 'react-redux';
-
-import {getCurrentUserId} from 'mattermost-redux/selectors/entities/users';
-import {getUserTimezone} from 'mattermost-redux/selectors/entities/timezone';
-import {getUserCurrentTimezone} from 'mattermost-redux/utils/timezone_utils';
-
-import {areTimezonesEnabledAndSupported} from 'selectors/general';
-
-import DateSeparator from './date_separator';
-
-function mapStateToProps(state, props) {
-    const currentUserId = getCurrentUserId(state);
-
-    let userTimezone;
-    if (props.userTimezone) {
-        userTimezone = props.userTimezone;
-    } else {
-        userTimezone = getUserTimezone(state, currentUserId);
-    }
-
-    return {
-        enableTimezone: areTimezonesEnabledAndSupported(state),
-        timeZone: getUserCurrentTimezone(userTimezone),
-    };
-}
-
-export default connect(mapStateToProps)(DateSeparator);
+export {default} from './date_separator';
```

Comment:
Why do we need this file? Can't we directly use `./date_separator` file?

Reply:
I thought i had an outstanding review on this one. Really minor nit anyway

Reply:
Thanks!

> Why do we need this file? Can't we directly use `./date_separator` file?

Yeah, 0/5 I think it's generally better to always have an index, but I could go either way.

Reply:
We've written that as
```javascript
import DateSeparator from './date_separator';
export default DateSeparator;
```
elsewhere, but I think that's because we didn't yet support export from if I remember correctly. I'm 0/5 on whether or not we move the file or reexport from an index file though. As long as we're using importing it as `components/post_view/date_separator` everywhere, it won't matter to any calling code